src/assets/scripts/client/UiController.js

```diff
@@ -73,7 +72,6 @@ class UiController {
         this.$element = $element;
 
         this.$airportList = this.$element.find(SELECTORS.DOM_SELECTORS.AIRPORT_LIST);
-        this.$airportListNotes = this.$element.find(SELECTORS.DOM_SELECTORS.AIRPORT_LIST_NOTES);
```

Comment:
Please remove AIRPORT_LIST_NOTES from SELECTORS and there should be also some HTML code in the template that is no longer needed.

Reply:
Still taking out the "is reliable" section from the airport selection because all airports that are going to be accepted have to be reliable.

Reply:
I have no idea what's the purpose of the reliability flag. Maybe @n8rzz or @erikquinn know?

Reply:
That me take a look at that quick.  chances are that isn't used anymore.  standby...

Reply:
@felixscheffer Could you review this again?

Reply:
Looks good to me. Nice work. 👍